lisa/testsuite.py

```diff
@@ -154,11 +154,17 @@ def handle_exception(
             self.set_status(TestStatus.PASSED, f"{phase}warning: {exception}")
         else:
             if self.runtime_data.ignore_failure:
-                log.info(f"case failed and ignored: {exception}")
+                log.info(
+                    f"case failed and ignored. "
+                    f"{exception.__class__.__name__}: {exception}"
+                )
                 self.set_status(TestStatus.ATTEMPTED, f"{phase}{exception}")
             else:
                 log.error("case failed", exc_info=exception)
-                self.set_status(TestStatus.FAILED, f"{phase}failed: {exception}")
+                self.set_status(
+                    TestStatus.FAILED,
+                    f"{phase}failed. {exception.__class__.__name__}: {exception}",
```

Comment:
Please put a space between phase and failed.

Reply:
The phase is optional, and a white space was appended to the phase at the beginning of this method, if it's not empty. So, if it's empty, there won't be a whitespace output. If there is phase, it will be like "deploy failed".